PEM - threshold_unit update as per PEM-4455 (#3244)
EnterpriseDB/docs: product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx

-   Use the `Target type` drop-down listbox to select the type of object that will be the focus of the alert.
-   Use the `Applies to server` drop-down listbox to specify the server type (EDB Postgres Advanced Server or PostgreSQL) to which the alert will be applied; you can specify a single server type, or `ALL`.
-   Use the `History retention` field to specify the number of days that the result of the alert execution will be stored on the PEM server.

> -   Use the `Threshold unit` field to specify the unit type of the threshold value. Any meaningful unit according to the output type of the underlying SQL statement is defined here. For example - `MB`, `#` (number), `Days`, `%`, `No of leader ID's` , `Null`, `No of worker errors`, `Hours`, `Minutes`, `State`, or anything meaningful.

Reviewer: Please add examples for various units that we have used in system alert templates in PEM. Below are some for your reference-
days -> A user expires in N days
# -> Function Count
MB -> Average table bloat in server
Hours -> Last Vaccum
Minutes-> Number of minutes lag of replica server from primary server
STATE -> Server Down
% -> CPU Utilization
'' -> Events lagging in one slony cluster
NULL -> Swap consumption
For NULL type create a note as well like it is only defined for some system alert templates and can not be used for User-defined alert templates

Include below types saying something like - we can have any other threshold units also according to our need for ex.
No of leader ID's-> PGD Group Raft Leader ID not matching
No of worker errors -> PGD worker error detected

Author: Updated the content as per the comment

-   Use the `Target type` drop-down listbox to select the type of object that will be the focus of the alert.
-   Use the `Applies to server` drop-down listbox to specify the server type (EDB Postgres Advanced Server or PostgreSQL) to which the alert will be applied; you can specify a single server type, or `ALL`.
-   Use the `History retention` field to specify the number of days that the result of the alert execution will be stored on the PEM server.

> -   Use the `Threshold unit` field to specify the unit type of the threshold value. Any meaningful unit according to the output type of the underlying SQL statement is defined here. For example following units are used in the system alert templates:

Reviewer 2: ```suggestion
-   Use the `Threshold unit` field to specify a unit type of the threshold value that corresponds to the output type of the underlying SQL statement. For example, the following units are used in the system alert templates:
```

  
  | Threshold unit | System Alert Template               |
  |----------------|------------------------------------ |
  | `#`            | Function count                      |
  | `%`            | CPU utilization                     |
  | `''`           | Events lagging in one slony cluster |
  | `days`         | A user expires in N days            |
  | `Hours`        | Last Vaccum                         |
  | `Minutes`      | Number of minutes lag of replica server from primary server |
  | `MB`           | Average table bloat in server       |
  | `NULL`         | Swap consumption                    |
  | `STATE`        | Server down                         |

  !!! Note

>       - `NULL` type is defined only for few system alert templates and cannot be defined for the custom alert templates.

Reviewer 2: ```suggestion
      - `NULL` type is only defined for a few system alert templates and cannot be defined for the custom alert templates.
```

>       - You can also define any meaningful `Threshold unit` like `No of leader ID's` is used for `PGD Group Raft Leader ID not matching` and `No of worker error` is used for `PGD worker error detected.

Reviewer 2: ```suggestion
      - You can also define any meaningful `Threshold unit` such as:
         - `No of leader ID's` for `PGD Group Raft Leader ID not matching` 
         - `No of worker error` for `PGD worker error detected.
```
